src/main/java/ProjectOcean/Controller/RemoveCourseFromSchedule.java

```diff
@@ -0,0 +1,7 @@
+package ProjectOcean.Controller;
+
+import ProjectOcean.Model.ICourse;
+
+public interface RemoveCourseFromSchedule {
```

Comment:
Add JavaDoc

Reply:
This interface is not used. I deleted it now.